rest-api/src/main/java/uk/gov/hmcts/ccd/definition/store/rest/endpoint/UserRoleController.java

```diff
@@ -39,6 +40,7 @@
         @ApiResponse(code = 205, message = "User role is updated successfully and the user agent SHOULD reset the document view"),
         @ApiResponse(code = 409, message = "Bad request, for example, incorrect data")
     })
+    @RouteToMasterDB
```

Comment:
on Repository layer things won't work. We could end up with inconsistencies. 
Imagine a controller uses 2 repositories Repo A and Repo B. Does a write with Repo A and a read with Repo B, and expects to see the write. 
We can annotate RepoB write method with @RouteToMasterDB. But we cannot annotate Repo B read method with @RouteToMasterDB. Because in general we want reads to go also to replicas. So we cannot reuse this method in other scenarios if we annotated it.
With this setup, the controller won't be able to see the write. And there will be inconsistencies.
Same issue for Service layers. A controller could use multiple Services